config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users do
+    resources :posts, only: [:index, :show], controller: 'posts'
+  end
```

Comment:
- [ ] Kindly make sure that your code only handles the 4 required URLs as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Make%20sure%20that%20you%20added%20code%20to%20handle%20only%20the%204%20required%20URLs.). Now you code allow all routes for the user.
![image](https://user-images.githubusercontent.com/24703171/221554726-1a8d4d4a-f4f9-4a90-977d-4f150d404827.png)